src/gap.c

```diff
@@ -791,6 +791,7 @@ Obj FuncWindowCmd (
   Int             n,  m;
   Int             i;
   Char *          ptr;
+  const Char *    inptr;
   Char *          qtr;
```

Comment:
It used to return `Char *`, but this PR makes it return a `const Char *`. This is in turn needed because it returns pointers to string constants.